torch/types.py

```diff
@@ -36,16 +38,16 @@
 _device = torch.device
 _qscheme = torch.qscheme
 _layout = torch.layout
-_size = Union[torch.Size, List[_int], Tuple[_int, ...]]
-_dispatchkey = Union[_str, torch._C.DispatchKey]
+_size = Union[torch.Size, List[builtins.int], Tuple[builtins.int, ...]]
```

Comment:
I wonder if it works without `builtins.` namespace.

```suggestion
_size = Union[torch.Size, List[int], Tuple[int, ...]]
```

Reply:
Let me try it out, I think that pyre doesn't run the code but only does static code analysis on top and probably confuse `_int` with a variable.